cos_registration_server/api/urls.py

```diff
@@ -9,4 +9,9 @@
     # make a version API url
     path("v1/devices", views.devices, name="devices"),
     path("v1/devices/<str:uid>", views.device, name="device"),
+    path(
+        "v1/devices/<str:uid>/foxglove_layouts/<str:layout_uid>",
```

Comment:
Wouldn't it be better for the layout_uid to be a query param ? `foxglove_layouts/some_name` vs `foxglove_layouts?layout=some_name`.

Reply:
No strong opinions but usually path are for resources on the server and parameters for performing action (download a file, connect to a remote, sort, filter).
The stack exchange doc seems to follow that: https://api.stackexchange.com/docs

Reply:
My (rough) understanding was that paths are for things that won't change while params are for things that may change (e.g. `v1` or `devices` will never change while the name of a layout may change).
That's why we need to put that on paper ^^